src/gisportal.js

```diff
@@ -775,6 +776,10 @@ gisportal.initStart = function()  {
    else if( gisportal.config.autoResumeSavedState == true && gisportal.hasAutoSaveState() )
       var autoLoad = function(){ gisportal.loadState( gisportal.getAutoSaveState() ); gisportal.launchMap();};
 
+   if(gisportal.autoLayer.url_get('wms_url') || gisportal.autoLayer.url_get('url_name')){
```

Comment:
is this still necessary since we are using event bindings to automatically load layers from URL params?